transcend/data_source_data_silo.go

```diff
@@ -14,8 +14,15 @@ func dataSourceDataSilo() *schema.Resource {
 		ReadContext: dataSourceDataSiloRead,
 		Schema: map[string]*schema.Schema{
 			"id": &schema.Schema{
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "The ID of the data silo. If not provided, the data silo will be discovered by other filters",
+			},
+			"link": &schema.Schema{
```

Comment:
Optional?

Reply:
I'm not sure, I think this is always filled in for data silos?

Reply:
Oh yeah [it's a resolver property](https://github.com/transcend-io/main/blob/048cd76abf0f1619fcd7333e4f4df9cf7125d30c/packages/backend/src/models/dataSilo/resolvers.ts#L123) so it's always present